libraries/chain/apply_context.cpp

```diff
@@ -288,6 +288,12 @@ void apply_context::execute_inline( action&& a ) {
       control.check_actor_list( actors );
    }
 
+   if( !privileged && control.is_producing_block() ) {
+      const auto& chain_config = control.get_global_properties().configuration;
+      EOS_ASSERT( a.data.size() < std::min(chain_config.max_inline_action_size, control.get_max_nonprivileged_inline_action_size()),
+                  inline_action_too_big_nonprivileged,
+                  "inline action too big for nonprivileged account ${account}", ("account", a.account));
+   }
    // No need to check authorization if replaying irreversible blocks or contract is privileged
```

Comment:
- Shouldn't we also be checking this in `execute_context_free_inline` ? 

- I think we should remove the checks in webassembly `interface::send_inline` & `interface::send_context_free_inline`. Although at this point that would require another replay test.

Reply:
Added to `execute_context_free_inline`.